<launch>
  <node name="pure_pursuite" pkg="agbot_nav" type="new_ppcontroller.py"/>
  <param name="file_name" value="waypoints_5_7_1.txt"/>
  <node pkg="base_level_control" type="base_level_control.py" name="base_level_control"/>
  <node pkg="TCP_command" type="TCPTest.py" name="TPCtoArduino"/>
  <param name="pidBypass" value="True"/>
  <node pkg="melonCV" type="real_time_object_detection.py" name="watermelon_Detector"/>
  <param name="melonBypass" value="True"/>
  <param name="maxPixel" value = "400"/>
  <param name="Kp_melon" value = "0.5"/>
  <param name="PP_restart" value = "False"/>
  <param name="bypassPedalPos" value = "0.4"/>
  <include file="$(find rosbridge_server)/launch/rosbridge_websocket.launch" >
  </include>
</launch>
